test_scripts/Defects/6_1/885_For_some_RPCs_Core_does_not_unsubscribe_from_softbuttons_after_receiving_a_response/UpdateTurnList.lua

```diff
@@ -0,0 +1,94 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/885
+-- Description:
+-- For some RPCs, Core does not unsubscribe from softbuttons after receiving a response
+-- Precondition:
+-- SDL and HMI are started
+-- In case:
+-- 1) App is registered and activated
+-- 2) App sends "UpdateTurnList" request
+-- Expected result:
+-- 1) For all of the popup-based RPCs which contain softButtons, Core is supposed to stop processing softButton events once a response is received for the RPC.
+-- Currently the only one that does this is ScrollableMessage, for all of the others Core stays subscribed to their softButtons after receiving a response.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function pTUpdateFunc(tbl)
+    local VDgroup = {
+        rpcs = {
+            UpdateTurnList = {
+                hmi_levels = { "BACKGROUND", "FULL", "LIMITED" }
+            }
+        }
+    }
+    tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = {"Base-4",
+    "NewTestCaseGroup"}
+end
+
+local function UpdateTurnList()
+    local params = {
+        softButtons = {
+            { type = "TEXT", softButtonID = 4, text = "text" }
+        }
+    }
+    local responseDelay = 3000
+    local cid = common.getMobileSession():SendRPC("UpdateTurnList", params )
+    EXPECT_HMICALL("Navigation.UpdateTurnList",{
+        appID = common.getHMIAppId(),
+        softButtons = {
+            { type = "TEXT", softButtonID = 4, text = "text" }
+        }
```

Comment:
Should tie this directly to `params.softButtons` rather than duplicating the values